src/Classes/ItemListControl.lua

```diff
@@ -42,9 +42,22 @@ local ItemListClass = newClass("ItemListControl", "ListControl", function(self,
 	end
 	self.controls.deleteUnused = new("ButtonControl", {"RIGHT",self.controls.deleteAll,"LEFT"}, -4, 0, 100, 18, "Delete Unused", function()
 		local delList = {}
+		local abyssalJewelsList = {}
+		local abyssalJewelsOwnedbyItemsList = {}
```

Comment:
Missed these from your debugging, I think.

Reply:
when you've too much time looking at your own code ...